### PR TITLE
Sample application to show list of mini apps

### PR DESCRIPTION
**Pull request by ts-leojoseph-r**
Sample application with table view to show list of mini apps. Tableview displays the following items,
* Mini app icon
* Mini app title
* Mini app description

### CODE REVIEW

File: Example/Images.xcassets/image_placeholder.imageset/Contents.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "2.png",
```
Comment (corycaywood) [inline Example/Images.xcassets/image_placeholder.imageset/Contents.json]: This is a placeholder image that will be shown before the icon has finished downloading?

File: Example/Info.plist [hunk 2]
```diff
@@ -2,6 +2,11 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSAppTransportSecurity</key>
+	<dict>
+		<key>NSAllowsArbitraryLoads</key>
```
Comment (corycaywood) [inline Example/Info.plist]: Setting this will allow `HTTP` connections? Is there a reason this is needed?

File: Example/ViewController.swift [hunk 3]
```diff
@@ -1,9 +1,88 @@
...
+                self.tableView.reloadData()
+                self.dismiss(animated: false, completion: nil)
+            case .failure(let error):
+                print(error.localizedDescription)
+                self.displayAlert(title: "Error", message: "Couldn't retrieve Mini App list, please try again later")
+            }
+        }
+    }
+
+    func decodeListingResponse(with dataResponse: Data?) -> [MiniAppInfo]? {
```
Comment (corycaywood) [inline Example/ViewController.swift]: I don't see anyplace where this function is used. Seems like something that shouldn't be needed?

File: Example/Images.xcassets/image_placeholder.imageset/Contents.json (hunk 1 above)
Comment (ts-leojoseph-r) [reply Example/Images.xcassets/image_placeholder.imageset/Contents.json]: Yes, Looks like the image name inside the json was wrong. I have updated in the new commit now.

File: Example/ViewController.swift (hunk 3 above)
Comment (ts-leojoseph-r) [reply Example/ViewController.swift]: Yes. Removed them now

File: Example/Info.plist (hunk 2 above)
Comment (ts-leojoseph-r) [reply Example/Info.plist]: No. It will allow only when NSExceptionAllowsInsecureHTTPLoads is added in NSAppTransportSecurity

Comment (donnie-jp) [reply Example/Info.plist]: Why do we need it? The backend doesn't have proper https / cert?

@corycaywood this is a workaround that allows all connections, including insecure ones.

Comment (ts-leojoseph-r) [reply Example/Info.plist]: I thought the staging is not gonna have one. Looks it is having valid one. Will update the plist.

File: MiniApp.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -17,6 +17,8 @@
 		607FACDB1AFB9204008FA782 /* Main.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 607FACD91AFB9204008FA782 /* Main.storyboard */; };
 		607FACE01AFB9204008FA782 /* LaunchScreen.xib in Resources */ = {isa = PBXBuildFile; fileRef = 607FACDE1AFB9204008FA782 /* LaunchScreen.xib */; };
 		607FACEC1AFB9204008FA782 /* Tests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 607FACEB1AFB9204008FA782 /* Tests.swift */; };
+		C3866B58983DE5A1C9FC45CF /* Pods_MiniApp_Tests.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 6E1CC4976FCCFA9C5737280C /* Pods_MiniApp_Tests.framework */; };
+		F8673656152B3624D9E6EAAB /* Pods_MiniApp_Example.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = FB348493F8CB435AB6C5F86C /* Pods_MiniApp_Example.framework */; };
```
Comment (donnie-jp) [inline MiniApp.xcodeproj/project.pbxproj]: We don't usually commit with Pods integrated but it's up to you if this is easier. There are pros/cons to both approaches.

Comment (ts-leojoseph-r) [reply MiniApp.xcodeproj/project.pbxproj]: Sure. Will consider this in the next commit.

**Pull request merged**